cypress/integration/common/index.js

```diff
@@ -198,11 +198,15 @@ And('there should be no {string} table in the document', (string) => {
 And(
 	'the Patient Information link to {string} appears on the page',
```

Comment:
this would change to `And('the {string} link to {string} appears on the page',(buttonText, patientInfoLink)=>{`